src/command-parser/parserConfig.ts

```diff
@@ -153,6 +148,50 @@ const opCertSigningArgs = {
   },
 }
 
+const msgSigningArgs = {
```

Comment:
We should also add the new call to `README.md` (and ideally also add a guide to `docs/`?)

Reply:
Done.